### PR TITLE
Fixed #23

### LINKED ISSUE #23: Server Crashed after /wp add {Player} || Crashdump
[19:45:38.655] [Server thread/CRITICAL]: Error: "Call to undefined method pocketmine\Server::getPlayer()" (EXCEPTION) in "plugins/WorldProtect_dev-39.phar/src/aliuly/worldprotect/WpProtectMgr" at line 76

--- Stack trace ---

#0 plugins/WorldProtect_dev-39.phar/src/aliuly/worldprotect/common/SubCommandMap(48): aliuly\worldprotect\WpProtectMgr->onSCommand(object ipad54\netherblocks\player\Player#98672, object pocketmine\command\PluginCommand#23889, string[3] add, string[19] Dragon's Apocalypse, array[1])

#1 plugins/WorldProtect_dev-39.phar/src/aliuly/worldprotect/common/BasicPlugin(123): aliuly\worldprotect\common\SubCommandMap

### PR DESCRIPTION
**Pull request by supercrafter333**

### CODE REVIEW

**Comment by supercrafter333** [comment]
@SOF3

File: src/aliuly/worldprotect/WpProtectMgr.php [hunk 1]
```diff
@@ -96,7 +96,7 @@ public function onSCommand(CommandSender $c,Command $cc,$scmd,$world,array $args
 					if ($this->owner->authCheck($world,$iusr)) {
 						$this->owner->authRm($world,$iusr);
 						$c->sendMessage(mc::_("[WP] %1% removed from %2%'s auth list",$i,$world));
-						$player = $this->owner->getServer()->getPlayer($i);
+						$player = $this->owner->getServer()->getPlayerExact($i);
```
Comment (SOF3) [inline src/aliuly/worldprotect/WpProtectMgr.php]: Isn't it supposed to be by prefix?

**Review by SOF3 — CHANGES_REQUESTED** [review]

File: src/aliuly/worldprotect/WpProtectMgr.php (hunk 1 above)
Comment (supercrafter333) [reply src/aliuly/worldprotect/WpProtectMgr.php]: Hmm... Yes?

**Comment by supercrafter333** [comment]
Now it's finished @SOF3

**Review by SOF3 — APPROVED** [review]

**Pull request merged**